Configs/.config/waybar/modules/style.css

```diff
@@ -3,7 +3,7 @@
     border-radius: 0px;
     font-family: "JetBrainsMono Nerd Font";
     font-weight: bold;
-    font-size: ${s_fontpx}px;
+    /* font-size: ${s_fontpx}px; disabled for multi monitor support */
```

Comment:
thank you, but the scaling is way off in my monitor when the font size is not specified!
I think this should be handled by `~/.config/waybar/config.ctl`, for example when col 2 is empty, then remove this line, else if size is specified in col 2, then use that font size.